### PR TITLE
Documentation: Explain data encoding and give examples

### PR DESCRIPTION
**Pull request by siro20**
Extend the physical layer documentation by using the signal description from README.md
and give example how data is encoded.

Reference physical layer markdown document from landing page.

Images are licensed under CC BY-SA.

Signed-off-by: Patrick Rudolph <rudolphpatrick05@gmail.com>

### CODE REVIEW

File: doc/PhysicalLayer.md [hunk 1]
```diff
@@ -1,8 +1,31 @@
 # Daikin P1/P2 bus physical layer
 
-The P1/P2 bus is based on the HBS standard.
-It's a low speed, half-duplex, two-wire power line communicaton (PLC) protocol.
+The P1/P2 bus is based on Japanese the Home Bus System (ET-2101), which is a low speed, half-duplex,
```
Comment (agittins) [inline doc/PhysicalLayer.md:3]:
```suggestion
The P1/P2 bus is based on the Japanese the Bus System (ET-2101), which is a low speed, half-duplex,
```

**Review by agittins — APPROVED** [review]
Great info!
(my approval doesn't count for anything, just an interested follower) :-)

File: doc/PhysicalLayer.md (hunk 1 above)
Comment (siro20) [reply doc/PhysicalLayer.md]: Done

**Comment by Arnold-n** [comment]
Thanks Patrick and Ashley!

**Pull request merged**